v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/templates/ProtegrityDataTokenization.java

```diff
@@ -219,7 +226,9 @@ public static PipelineResult run(ProtegrityDataTokenizationOptions options,
       DataflowPipelineOptions dataflowOptions) {
     SchemasUtils schema = null;
     try {
-      schema = new SchemasUtils(options.getDataSchemaGcsPath(), StandardCharsets.UTF_8);
+      if (options.getDataSchemaGcsPath() != null) {
+        schema = new SchemasUtils(options.getDataSchemaGcsPath(), StandardCharsets.UTF_8);
+      }
```

Comment:
Why did you add this null checking? Is it connected with some appeared warnings?

Reply:
If `options.getDataSchemaGcsPath()` will be null, what we want to do? We want to LOG about it, we want to stop execution, we want to have some default value in that case?

Reply:
Ok, now if this property is null, throws NPE in SchemasUtils's constructor because there are transforms with this parameter.
So I suppose we should check it before and show correct message for user.

Reply:
Actually, we have a catch and check a bit lower that are supposed to validate the schema presence. But you've got a point, let's refactor this part like this:
```java
checkArgument(options.getDataSchemaGcsPath() != null, "Data schema is mandatory.");
SchemasUtils schema = null;
try {
  schema = new SchemasUtils(options.getDataSchemaGcsPath(), StandardCharsets.UTF_8);
}...
```

Reply:
Thanks, fixed. Now use checkArgument.